### ISSUE TITLE
Responsive 

### ISSUE THREAD

**Issue by vtxmg** [loopj/jquery-tokeninput#791]
Its listing div is fixed in width, I want responsive ... anybody have idea

**Comment by gg4u** [comment]
I agree, a documentation on how to make div inherit or apply relative width is a must.
Examples of fixed width of 400px stretch over the viewport of mobile handsets (at least in console preview).
I would be happy if I cannot avoid customising with media queries on css, because I would like to apply same classes of other libriaries. (materializecss is currently my favourite).

**Comment by gg4u** [comment]
I solved without css, with one line you can fix responsiveness of dropdown list.

in `jquery.tokeninput.js` modify the` show_dropdown()` function adding css width:

```
    function show_dropdown() {
        dropdown
            .css({
                // FIX for RESPONSIVE LAYOUT
                width: $(token_list)[0].offsetWidth,
                
                position: "absolute",
                top: $(token_list).offset().top + $(token_list).outerHeight(),
                left: $(token_list).offset().left,
                zindex: 999
            })
            .show();
    }

```

this set the width of dropdown to computed width of the form to <ul.token-input-list-[your template]> element: you can skip fixed width on CSS and media query nightmare.

If you want to match computed list of another custom element you can also use:


               `width: document.getElementById("yourElement_ID").offsetWidth,`
